pyproject.toml

```diff
@@ -18,7 +18,7 @@ matplotlib = [
     { version = "^3.5.0", markers = "python_version ~= '3.7'"},
     { version = "^3.6.0", markers = "python_version ~= '3.11.0'"},
 ]
-fastapi = "^0.92"
+fastapi = ">=0.92,<1.0.0"
```

Comment:
I think `~0` could work here according to [docs](https://python-poetry.org/docs/dependency-specification/#tilde-requirements), but hurts readability.

Reply:
Yes, I agree. 👍